packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -714,8 +713,8 @@ const runOptimize = async () => {
 	};
 
 	// InferredParameters is to link a calibration run to this optimize call.
-	if (inferredParameters.value) {
-		optimizePayload.extra.inferredParameters = inferredParameters.value[0];
+	if (modelConfiguration.value) {
+		optimizePayload.extra.inferredParameters = modelConfiguration.value.simulationId;
```

Comment:
What is sent if this simulationId has not been filled in by running a calibration?
pyciemss-service will accept this as an optional field and will provide None if it is not filled in:

https://github.com/DARPA-ASKEM/pyciemss-service/blob/main/service/models/operations/optimize.py

https://github.com/DARPA-ASKEM/pyciemss-service/blob/main/service/utils/tds.py#L192-L194